### DISCUSSION TITLE
remove Multi-line chart plugin

### DISCUSSION THREAD

**Discussion by junlincc** [apache/superset#12721] [General]
Multi-line chart seems broken again(after few fixes). 

Wonder if we can remove this plugin completely?
Reason:
1. It was broken for a long time before, no one reported during that period, that tells me this chart might not be used often. 
2. User can create the same chart using Dual Line Chart
3. It's extremely hard to maintain and fragile

**Comment by rusackas** [comment]
Please! There's a workaround, and if we need new/different features, I'd far rather invest in ECharts than keep this on life support.
